pkgs/top-level/all-packages.nix

```diff
@@ -3873,6 +3873,8 @@ with pkgs;
 
   easycrypt-runtest = callPackage ../applications/science/logic/easycrypt/runtest.nix { };
 
+  EasyOCR = with python3.pkgs; toPythonApplication EasyOCR;
```

Comment:
Nit: can we make this top-level attribute name lower-case please?

Rationale: it seems pretty common to have package names in lisp-case, and it makes `nix run nixpkgs#easyocr` work without having to set meta.mainProgram.